webapp/wmservices/src/main/java/org/apache/oodt/cas/wmservices/services/WMJaxrsServiceV2.java

```diff
@@ -78,6 +79,50 @@ public WorkflowManagerStatus getWorkflowManagerStatus() {
     }
   }
 
+  /**
+   * returns all registered events
+   *
+   * @return events
+   */
+  @GET
+  @Path("events")
+  @Produces({MediaType.APPLICATION_JSON, MediaType.APPLICATION_XML})
+  public WorkflowEventListResource getRegisteredEvents() {
+    try {
+      WorkflowManagerClient wmclient = getContextClient();
+      List events = wmclient.getRegisteredEvents();
+      WorkflowEventListResource eventResource = new WorkflowEventListResource(events);
+      return eventResource;
+    } catch (WorkflowException e) {
```

Comment:
Lets use multiple-catch in single line here because same body content is repeating.